src/api/app.js

```diff
@@ -46,3 +65,45 @@ app.get('/', function (req, res) {
 
 const port = 3000;
 app.listen(port, () => logger.info(`Start service on port ${port}!`));
+
+function removeEmailIfOlderThan(email, dateTimeBoundary) {
+    return email["datetime"] > dateTimeBoundary;
+}
+
+function filterByEmail(email, to) {
+    let actualToEmail = email["personalizations"][0]["to"][0]["email"];
```

Comment:
The check is just working for the first provided "to"-email address, maybe we can extend the check to check all provided emails?